MakairaConnect.php

```diff
@@ -47,9 +48,29 @@ public function build(ContainerBuilder $container)
      */
     public function install(InstallContext $installContext)
     {
+        $this->checkSchemaStructure();
         $this->installModels();
     }
 
+    public function update(UpdateContext $context)
+    {
+
+        $this->checkSchemaStructure();
+        parent::update($context);
+    }
+
+    private function checkSchemaStructure()
+    {
+        $conn = $this->container->get('dbal_connection');
+        $state = $conn->query(
+            "SELECT SCHEMA_NAME FROM information_schema.SCHEMATA WHERE SCHEMA_NAME = 'mak_revision' LIMIT 1"
```

Comment:
I tested on my local(by executing the query directly in Mysql) and it didn't get me the correct result. I tried the following query and it worked:
```suggestion
            "SELECT table_name FROM information_schema.tables where table_name = 'mak_revision' LIMIT 1"
```

Reply:
I already fixed this, but in the wrong folder.